src/game/clock.h

```diff
@@ -3,6 +3,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+void Clock_ToggleTurbo(void);
```

Comment:
This function no longer Toggles things, it Cycles things. It also no longer toggles turbo "mode" (which would be OK to omit) but cycles turbo speed multipliers. So it should be named `Clock_CycleTurboSpeed` (we omit the "Mul" part here because it's an implementation detail).